packages/components/src/local/molecules/channel-coa-message-detail-2/helpers/visibility.ts

```diff
@@ -37,19 +38,19 @@ const isThisChannel = (channel: ChannelData, channelType: SpecialChannelTypes):
 /** whether this user can see the collaborative aspects of this channel */
 export const userCanSeeCollab = (_channel: ChannelData, _role: ForceRole): boolean => {
   const parts = _channel.participants
-  return !!parts.find((part: Participant) => {
+  return !!parts.find((part: CoreParticipant) => {
```

Comment:
If you switch to `ParticipantCollab` then the below logic can still be done using the `viewUnreleasedVersions` property.